opencti-platform/opencti-graphql/src/domain/grant.js

```diff
@@ -16,7 +18,19 @@ export const addRole = async (context, user, role) => {
     assoc('description', role.description ? role.description : ''),
     dissoc('capabilities'),
   )(role);
-  const { element, isCreation } = await createEntity(context, user, roleToCreate, ENTITY_TYPE_ROLE, { complete: true });
+
+  let completeRoleToCreate;
+  if(isFeatureEnabled(PROTECT_SENSITIVE_CHANGES_FF)) {
+      completeRoleToCreate = {
+        ...roleToCreate,
+        is_sensitive_changes_allow: role.is_sensitive_changes_allow ?? true, //default when undefined is true
```

Comment:
false by default ? otherwise all new roles created will have this option by default